cmake/modules/XROOTDApplePostInstall.cmake

```diff
@@ -0,0 +1,43 @@
+# Post install routine needed to change the LC_RPATH variable of the XRootD
+# libraries on macOS after installation. The function changes the value from
+# ${build_libdir} to ${install_libdir} with the `install_name_tool` executable.
+# On successive reiterations of the `cmake --install` command,
+# `install_name_tool` would error out since LC_RPATH would be already set to
+# ${install_libdir}. Since that is what we want in the end, we discard these
+# errors.
+
+function(xrootd_libs_change_rpath build_libdir install_libdir)
+
+  set(XROOTD_ALL_LIBRARIES libXrdAppUtils.dylib libXrdBlacklistDecision-4.so
```

Comment:
Can we glob at this point?

Reply:
Ok if it's too much meat I'll just put everything in a GLOB :smile: